app/models/taxon.rb

```diff
@@ -19,6 +19,8 @@ class Taxon < ActiveRecord::Base
   # Skip the more onerous callbacks that happen after grafting a taxon somewhere else
   attr_accessor :skip_after_move
 
+  attr_accessor :skip_after_activate
```

Comment:
Is this actually getting used or was it just copied from `skip_after_move` / `handle_after_move`?

Reply:
I guess thats not being used, I can remove it